rocketpool/node/node.go

```diff
@@ -154,23 +155,41 @@ func run(c *cli.Context) error {
 
 	// Run task loop
 	go func() {
+		// we assume clients are synced on startup so that we don't send unnecessary alerts
+		wasExecutionClientSynced := true
+		wasBeaconClientSynced := true
 		for {
```

Comment:
you found my favorite tech debt!

Reply:
ha. I found it odd that it waits since the system has to be able to detail with out-of-sync clients at all times anyway. If you want I can clean this up but I'd prefer to do it in a separate PR after this one just to keep them independent and simple.

Reply:
nah don't sweat it. i think joe might have fixed it on a branch.

it's on my mental backlog to replace this for loop with a time.Ticker-based abstraction, if he didn't, though